segnet/train/train_segnet.py

```diff
@@ -22,6 +23,7 @@ def train_unet(
     them into training and validation, while saving the best model trained.
 
     Args:
+        model (tensorflow.python.keras.engine.training.Model): A keras model instance.
```

Comment:
I think we can reduce the type to `tensorflow.keras.Model` which is its real name.

Reply:
@gmagannaDevelop I'll rename it later, no worries.